components/dashboard/src/service/service.ts

```diff
@@ -4,25 +4,14 @@
  * See License-AGPL.txt in the project root for license information.
  */
 
-import { GitpodClient, GitpodServer, GitpodServerPath, GitpodService, GitpodServiceImpl, User, WorkspaceInfo } from '@gitpod/gitpod-protocol';
+import { GitpodClient, GitpodServer, GitpodServerPath, GitpodService, GitpodServiceImpl, User } from '@gitpod/gitpod-protocol';
 import { WebSocketConnectionProvider } from '@gitpod/gitpod-protocol/lib/messaging/browser/connection';
 import { createWindowMessageConnection } from '@gitpod/gitpod-protocol/lib/messaging/browser/window-connection';
 import { JsonRpcProxy, JsonRpcProxyFactory } from '@gitpod/gitpod-protocol/lib/messaging/proxy-factory';
 import { GitpodHostUrl } from '@gitpod/gitpod-protocol/lib/util/gitpod-host-url';
 import { log } from '@gitpod/gitpod-protocol/lib/util/logging';
-import React from 'react';
 
-export interface Service {
-    user?: User;
-    workspaces: WorkspaceInfo[];
-    service: GitpodService;
-}
-
-export class SimpleServiceImpl implements Service {
-    user?: User;
-    service = createGitpodService();
-    workspaces: WorkspaceInfo[] = [];
-}
+export const gitpodHostUrl = new GitpodHostUrl((window as any).PREVIEW_URL || window.location.toString());
```

Comment:
I don't think we should build such a variance into the code. It should always be `window.location` or manually changed here (and not checked in).